lib/matplotlib/backends/backend_ps.py

```diff
@@ -1006,28 +985,18 @@ def print_figure_impl(fh):
 
     @cbook._delete_parameter("3.2", "dryrun")
     def _print_figure_tex(
-            self, outfile, format, dpi, facecolor, edgecolor,
-            orientation, papertype, *,
-            metadata=None, dryrun=False, bbox_inches_restore=None, **kwargs):
```

Comment:
So this function is technically private, but we still have `_delete_parameter` on it, so is it really safe to be deleting other parameters here?

Reply:
The _delete_parameter is there because previously it couldn't go onto the public caller (print_ps) which just accepted dryrun into kwargs (xref https://github.com/matplotlib/matplotlib/pull/16231).
print_ps only ever passes facecolor/edgecolor as kwargs anyways, so they would get silently absorbed into kwargs.